### ISSUE TITLE
Disable touch screen using front face sensors, Motorola XT701 (and others)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BinaryBlue/imsdroid#36]
```
What steps will reproduce the problem?
1.  When using the program, the touchscreen on my phone is still active.
2.  This can lead to any number of problems when touching the phoine to my ear 
or face.
3.  Can you disable the touchscreen while the sensor says the phone is near my 
face?

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?

IMSDroid 1,0,263  Android 2.1
Please provide any additional information below.


```

Original issue reported on code.google.com by `jerald...@gmail.com` on 20 Aug 2010 at 4:41

**Comment by GoogleCodeExporter** [comment]
Original comment by `boss...@yahoo.fr` on 20 Aug 2010 at 6:59
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This makes the app totally unusable for me, which is a shame as this is the 
only voip app to support SMS Gateways.  Suggest it gets a higher priority, as 
this probably affecting a lot of users?
```

Original comment by `jerald...@gmail.com` on 24 Aug 2010 at 11:30

**Comment by GoogleCodeExporter** [comment]
```
I wish I was a coder, I would contribute for sure, but I am a lowly user.  :-K

Super project though!
```

Original comment by `jerald...@gmail.com` on 24 Aug 2010 at 11:49

**Comment by GoogleCodeExporter** [comment]
```
As far as I know the project is free of charge.
You have posted your bug 3 days ago and I haven't got time to review the issue. 
You should know that I'm not paid to work on the project and I do it in my 
spare time.
What is a shame is to post the same issue on SipDroid forum 
(http://code.google.com/p/sipdroid/issues/detail?id=609#c0) at the same time (3 
days ago) by saying that they have nicer interface blahblahblah....
FYI: A have changed the level from "Medium" to "Low".
Good Luck.
```

Original comment by `boss...@yahoo.fr` on 24 Aug 2010 at 6:59
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Wow.  Uninstalling permanently.  Posting this to the reviews of the app.
```

Original comment by `jerald...@gmail.com` on 24 Aug 2010 at 7:40

**Comment by GoogleCodeExporter** [comment]
Original comment by `boss...@yahoo.fr` on 24 Aug 2010 at 7:44
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
```
The issue is closed but will remain visible. You can add the URL in your review.
Here is the URL: http://code.google.com/p/imsdroid/issues/detail?id=36.
```

Original comment by `boss...@yahoo.fr` on 24 Aug 2010 at 7:56

**Comment by GoogleCodeExporter** [comment]
```
Can't stand the truth eh?  If they add sms gateway this project is toast.
```

Original comment by `jerald...@gmail.com` on 24 Aug 2010 at 8:01

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.0.279.
```

Original comment by `boss...@yahoo.fr` on 7 Sep 2010 at 8:15
- Changed state: **Fixed**
